### ISSUE TITLE
[eBUG] NPEs at startup on master following recent changes

### ISSUE THREAD

**Issue by tychobrailleur** [akasolace/HO#392]
**Describe the bug**
App on `master` throws a lot of `NullPointerException` at startup, showing in the logs as:

```
 [Error]   Lineup: getHeimspiel: java.lang.NullPointerException
 [Error]   Lineup: getHeimspiel: java.lang.NullPointerException
 [Error]   Lineup: getHeimspiel: java.lang.NullPointerException
 [Error]   Lineup: getHeimspiel: java.lang.NullPointerException
 [Error]   Lineup: getHeimspiel: java.lang.NullPointerException
 [Error]   Lineup: getHeimspiel: java.lang.NullPointerException
```

Looking into this a bit, this seems to be related to changes for #299  in `Lineup#getUpcomingMatch()` which makes a call to `MatchKurzInfo.isNeutral()`:

```
	public final boolean isNeutral(){return this.m_iIsNeutral.equals(Boolean.TRUE);}
```

where `this.m_iIsNeutral` is `null`. 

My assumption here is that the matches were already downloaded before the db change, and the `isNeutral` column wasn't set.  It's probably a corner case we'd need to address, if that's the case.

(Incidentally we should probably update the log message to refer to `getUpcomingMatch` rather than `getHeimspiel`)

**To Reproduce**
Run recent code on `master`.

**Comment by akasolace** [comment]
@wsbrenk those are the error messages I was referring to in my comment on your PR. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by wsbrenk** [comment]
@tychobrailleur thanks - i will repair isNeutral and isDerby

**Comment by akasolace** [comment]
closed by #393

**Comment by tychobrailleur** [comment]
The fix adds a null check, but shouldn't we address the case where users already have matches downloaded, and installing the new version? The home / derby features won't work for those matches—maybe not a big deal, but at leat we should document?
